acceptance/tests/fixtures/bases/api-gateway-v2/apigateway.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+apiVersion: mesh.consul.hashicorp.com/v2beta1
+kind: APIGateway
+metadata:
+  name: gateway
+spec:
+  gatewayClassName: gateway-class
+  listeners:
+  - protocol: HTTP
```

Comment:
I think for now we can just have the tcp listener since we're not implementing any of the http listener/route functionality just yet